src/module_loader.rs

```diff
@@ -14,19 +14,51 @@ use std::{
     rc::Rc,
 };
 
+#[allow(unused_variables)]
+#[cfg(feature = "custom_import")]
+pub trait ImportProvider {
+    fn resolve(
+        &mut self,
+        specifier: &str,
+        referrer: &str,
+        kind: deno_core::ResolutionKind,
+    ) -> Result<ModuleSpecifier, anyhow::Error> {
+        Ok(deno_core::resolve_import(specifier, referrer)?)
+    }
+    fn import(
+        &mut self,
+        specifier: ModuleSpecifier,
+        referrer: Option<ModuleSpecifier>,
+        is_dyn_import: bool,
+        requested_module_type: deno_core::RequestedModuleType,
+    ) -> Result<String, anyhow::Error> {
+        return Err(anyhow!(

```

Comment:
If you just want to deny all you're better off just passing none instead of an ImportProvider surely?